parser/sql/dialect/mysql/src/main/antlr4/imports/mysql/DDLStatement.g4

```diff
@@ -207,7 +207,7 @@ truncateTable
 
 createIndex
     : CREATE createIndexSpecification? INDEX indexName indexTypeClause? ON tableName keyListWithExpression indexOption?
-    (alterAlgorithmOption | alterLockOption)*
+    alterAlgorithmOption? alterLockOption?
```

Comment:
Please refer to MySQL syntax.
https://github.com/mysql/mysql-server/blob/1bfe02bdad6604d54913c62614bde57a055c8332/sql/sql_yacc.yy#L9089